src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/CockroachDBTranslator.java

```diff
@@ -105,6 +107,18 @@ public String translate(final Cast cast) {
         return cast.isEnclosed() ? "(" + translation + ")" : translation;
     }
 
+    @Override
+    public String translate(final Concat concat) {
+        inject(concat.getDelimiter());
+        inject(concat.getExpressions());
+
+        final String concatWs = format(" || %s || ", concat.getDelimiter().translate());
+
+        return concat.getExpressions().stream()
+                     .map(Expression::translate)
+                     .collect(Collectors.joining(concatWs));
+    }
+
```

Comment:
Why?
CockroachDB should support CONCAT_WS just fine

Reply:
CockroachDB is having issues in the tests with CONCAT_WS.

Reply:
Fixed!